### DISCUSSION TITLE
Are header spans possible?

### DISCUSSION THREAD

**Discussion by chbert** [tzezar/datagrid#38] [Q&A] [answered]
Hi @tzezar, first thank you for sharing this great project!
Quick question, does the library support header spans (multiple header rows)?

**Comment by tzezar** [comment, marked as answer]
Hi! The current version doesn't support this feature yet. However, I'm working on an official release that will include additional features, including support for header groups. Stay tuned!
![image](https://github.com/user-attachments/assets/7801399b-9629-4deb-9d4d-6eabc42fd0ac)

**Reply by chbert** [reply]
@tzezar great to hear! Let me know if we can support you in any way!
